<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QDialog" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_logo">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>280</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
border-bottom:1px solid #707172;
background: transparent;
font: 12pt &quot;Century Gothic&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
border-bottom:1px solid #707172;
background: transparent;
font: 12pt &quot;Century Gothic&quot;;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>161</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
font: 28pt &quot;Century Gothic&quot;;</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>420</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
#pushButton{
color:white;
border-radius:20px;
font: 14pt  'Century Gothic';
background:rgb(175, 170, 200);
}

#pushButton:hover{
border-radius:20px;
font: 14pt  'Century Gothic';
background:#E8E8E8;

color:black;

}</string>
   </property>
   <property name="text">
    <string>connecter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_logo_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1241</width>
     <height>631</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
   </property>
   <property name="text">
    <string>Powered by</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_logop">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>10</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>530</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_eye">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>300</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_eye">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_logo_3</zorder>
  <zorder>username</zorder>
  <zorder>password</zorder>
  <zorder>label_9</zorder>
  <zorder>label_logo</zorder>
  <zorder>label_2</zorder>
  <zorder>label_logop</zorder>
  <zorder>label</zorder>
  <zorder>label_eye</zorder>
  <zorder>pushButton</zorder>
  <zorder>pb_eye</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
